torch/csrc/autograd/python_variable.cpp

```diff
@@ -364,6 +364,54 @@ static PyObject* THPVariable_make_subclass(PyObject* _ignored, PyObject* args, P
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject* THPVariable_make_wrapper_subclass(PyObject* _ignored, PyObject* args, PyObject* kwargs) {
```

Comment:
nit. Write it as follows to avoid unused variable warnings
```suggestion
static PyObject* THPVariable_make_wrapper_subclass(PyObject* , PyObject* args, PyObject* kwargs) {
```